src/Build.UnitTests/FileLogger_Tests.cs

```diff
@@ -448,14 +448,14 @@ public void DistributedLoggerBadPath()
                                                           + "mylogfile.log");
 
                 fileLogger.Initialize(new EventSourceSink());
-                Assert.True(
-                              string.Compare(
-                                             fileLogger.InternalFilelogger.Parameters,
-                                             ";ShowCommandLine;logfile="
-                                             + Path.Combine(
-                                                            Directory.GetCurrentDirectory(),
-                                                            Path.DirectorySeparatorChar + "DONTEXIST" + Path.DirectorySeparatorChar + "mylogfile2.log"),
-                                             StringComparison.OrdinalIgnoreCase) == 0);
+
+                Assert.Equal(
+                    fileLogger.InternalFilelogger.Parameters,
+                    ";ShowCommandLine;logfile="
+                        + Path.Combine(
+                        Directory.GetCurrentDirectory(),
+                        Path.DirectorySeparatorChar + "DONTEXIST" + Path.DirectorySeparatorChar + "mylogfile2.log"), 
```

Comment:
Doesn't the leading `DirectorySeparatorChar` make that part an absolute path, making the combine useless? Could this just be `Path.Combine( Directory.GetCurrentDirectory(), "DONTEXIST", "mylogfile2.log")`?